packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -135,6 +161,31 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>, ToolbarItemS
       </ElementType>
     )
 
+    const submenu = menuOpen ? (
+      <>
+        <Ref innerRef={this.menuRef}>
+          <Popper align="start" position="above" targetRef={this.itemRef}>
+            {ToolbarMenu.create(menu, {
+              overrideProps: predefinedProps => ({
```

Comment:
```suggestion
              overrideProps: (predefinedProps: ToolbarItemProps) => ({
```

Reply:
Are we going to move `overrideProps` to a class method?

Reply:
I don't think so. This is the only place where it is called, just once.